pkg/kotsutil/kots.go

```diff
@@ -162,6 +163,47 @@ func (k *KotsKinds) HasPreflights() bool {
 	return len(k.Preflight.Spec.Analyzers) > 0
 }
 
+func HasStrictPreflights(preflight *troubleshootv1beta2.Preflight) bool {
+	if preflight == nil {
+		return false
+	}
+	for _, a := range preflight.Spec.Analyzers {
+		if (a.ClusterVersion != nil && a.ClusterVersion.Strict.BoolOrDefaultFalse()) ||
```

Comment:
i still believe that this can be easily abstracted instead of checking for each individual analyzer

Reply:
which would be great if we add new analyzers later

Reply:
discussed in slack, to be updated